usolitaire/card.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+
+SUIT_SYMBOLS = {
+    "spades": "♠",
+    "diamonds": "♦",
+    "clubs": "♣",
+    "hearts": "♥",
+}
+
+class Card(object):
+    def __init__(self, rank: str, suit: str, face_up: bool = False):
```

Comment:
Can we add a new type Rank which will be a Literal of all the possible ranks?
And something similar for Suit?
And then use those types on the other files as well?